openfisca_web_api/loader/variables.py

```diff
@@ -59,23 +59,22 @@ def build_formula(
 
 
 def build_formulas(
-    formulas, country_package_metadata, source_file_path, tax_benefit_system
+    formulas, country_package_metadata, source_file_path
 ):
     return {
         start_date: build_formula(
-            formula, country_package_metadata, source_file_path, tax_benefit_system
+            formula, country_package_metadata, source_file_path
         )
         for start_date, formula in formulas.items()
     }
 
 
-def build_variable(variable, country_package_metadata, tax_benefit_system):
+def build_variable(variable, country_package_metadata, ):
```

Comment:
Is it intentional to have a trailing comma?
```suggestion
def build_variable(variable, country_package_metadata):
```